app/poros/question.rb

```diff
@@ -10,10 +10,15 @@ def initialize(data)
     @question_number = data["id"]
     @question_text = fix_question(data["question_text"])
     @answer = data["correct_answer"]
-    @options = data["options"].shuffle
+    @options = complete_options(data).shuffle
   end
 
   def fix_question(data)
     data.sub(/^\d+\.\s/, "")
   end
+
+  def complete_options(data)
+    data["options"] << data["correct_answer"] if data["options"].length < 4
```

Comment:
This may result in a duplicate answer option in some cases

Reply:
Yes, let me rephrase, to check if the array includes the correct option!